docker/docker-compose.yaml

```diff
@@ -97,6 +97,45 @@ services:
     networks:
       - von
 
+  react-holder:
```

Comment:
I know it's not really part of this PR but could we change all `licensee` to `holder` for consistency (like in cloudagent-licensee definition above)?

Reply:
:+1: thanks for flagging this one up, it was on my list when I started working on this last week